Index: app/src/main/java/com/example/myapplication/MemberModel.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package com.example.myapplication\r\n\r\nclass MemberModel (\r\n    var docId: String? = null,\r\n    var email: String? = null,\r\n    var nickname: String? = null,\r\n    var deviceId: String? = null\r\n    )
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/com/example/myapplication/MemberModel.kt b/app/src/main/java/com/example/myapplication/MemberModel.kt
--- a/app/src/main/java/com/example/myapplication/MemberModel.kt	(revision 138f5e0bbdb2e4feffd1dd27e38b0e56bd42b0be)
+++ b/app/src/main/java/com/example/myapplication/MemberModel.kt	(date 1688105682697)
@@ -1,6 +1,6 @@
 package com.example.myapplication
 
-class MemberModel (
+data class MemberModel (
     var docId: String? = null,
     var email: String? = null,
     var nickname: String? = null,
